certs/ca/ca.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
you may get an email from github complaining about this file

just a random cert so /shrug